automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/user/SignInPage.scala

```diff
@@ -65,7 +65,11 @@ class SignInPage(val baseUrl: String)(implicit webDriver: WebDriver) extends Fir
 class GoogleSignInPopup(implicit webDriver: WebDriver) extends WebBrowser with WebBrowserUtil {
 
   def awaitLoaded(): GoogleSignInPopup = {
-    await text "to continue to"
+    try {
+      await text "to continue to"
+    } catch {
+      case e: TimeoutException => // ignore because text may or may not appears
+    }
```

Comment:
This seems like a logic error. We should know whether or not we expect that text to appear.

Reply:
Maybe.
methods calling chains are: `beginSignIn`=> `signIn`.
in `signIn` method, popup.singIn(email, password) handles accounts selection if detected. but execution fails before it gets a chance because this `wait text` time out.

Reply:
This appears to ultimately evaluate to a no-op.  Why have `awaitLoaded()` at all, after this change ?

Reply:
not sure I get the question. It's still needed in case text decided to show up. If taken it out, test may fail.

Reply:
completely agree with you. I want something that's going to work for either version. It's why I catch and ignore timeout exception instead of delete or change it.

Reply:
Instead of catching and ignoring the `TimeoutException`, please do something like this:
```
await condition (text("to continue to").findElement.isDefined || text("...").findElement.isDefined)
```
I don't know what the other text should be but you should be able to get it from a failure screenshot or HTML.